shared-keycloak-app.tf

```diff
@@ -0,0 +1,46 @@
+module "shared_keycloak" {
```

Comment:
this naming feels weird to me, it implies that there will be other keycloak configs in this module

this module should be representing the single keycloak module that will ever exist for the company

Reply:
Agree, I dont think the "shared" thing is needed

Reply:
updated to keycloak